src/ripple/app/misc/tests/AccountTxPaging.test.cpp

```diff
@@ -0,0 +1,246 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+#include <ripple/app/data/DatabaseCon.h>
+#include <ripple/app/misc/impl/AccountTxPaging.h>
+#include <beast/cxx14/memory.h>  // <memory>
+#include <beast/unit_test/suite.h>
+#include <cstdlib>
+#include <vector>
+
+namespace ripple {
+
+struct AccountTxPaging_test : beast::unit_test::suite
+{
+    std::unique_ptr<DatabaseCon> db;
+    NetworkOPs::AccountTxs accountTxs;
+    RippleAddress account;
+
+    void
+    run() override
+    {
+        std::string fixturesPath = std::getenv("TEST_FIXTURES");
```

Comment:
`std::getenv` returns null if the environment variable is not found. This will segfault the test.